### DISCUSSION TITLE
Booting problems (external midi device connected over USB)

### DISCUSSION THREAD

**Discussion by BobanSpasic** [probonopd/MiniDexed#264] [Q&A]
If I turn on the MiniDexed before I connect my music keyboard (over USB-To Host) - it is working like it should.
If I have my keyboard connected (either turned on or off) and start my MiniDexed after that - here comes to the problems. Either do not boot at all, or it takes a veeeeery long time etc. I can only see white boxes on the screen, sometime in the both rows, sometime just one row.

What would you need from me to test for a better bug repport?
Are there some log files stored on the SD Card that I can send to you?

**Comment by probonopd** [comment]
Power Supply issues? Try another (more Ampere) power supply.

Also, connect a HDMI screen and watch for any errors.

**Reply by BobanSpasic** [reply]
The keyboard (CME U-key v2) has its own power supply, it should not be an issue. MiniDexed is on a Samsung smartphone charger. 
I have had it once on HDMI in a such situation, and there were no errors, but also no "MiniDexed" text on the LCD screen and also no sound.
If it were about the power supply, it wouldn't work in any situation, but it is working if I connect the keyboard after the MiniDexed already started.
Is there a way to get some debug log from MiniDexed? Perhaps the same log as on the HDMI display?

**Reply by probonopd** [reply]
Could not hurt to try another power supply nevertheless.
When there are only withe boxes on the LCD, can you play on the keyboard and it produces sound? Or is the whole thing not working?

**Reply by BobanSpasic** [reply]
I'll test it again tomorrow, but as I can recall there was no sound.

**Reply by probonopd** [reply]
Did you do this?

> If the system seems to become unresponsive after a few seconds, remove `usbspeed=full` from `cmdline.txt` and repeat ([details](https://github.com/probonopd/MiniDexed/issues/39))

**Reply by anycolorsmith** [reply]
> The keyboard (CME U-key v2) has its own power supply, it should not be an issue. 

I run with CME witout external power, USB only. Works correct.
